src/bpmncwpverify/core/bpmn.py

```diff
@@ -34,8 +35,12 @@ def __init__(self, element: Element) -> None:
 
         self.name: str = element.attrib.get("name", self.id)
 
+    @classmethod
+    def from_xml(cls: Type[T], element: Element) -> T:
```

Comment:
Why have a `from_xml` method if there is a constructor for every element that takes an `Element` type? So if every element has a constructor that takes an XML tree as input, then what purpose does `from_xml` serve? 

I think we need the separate out the XML parsing from the object initialization. All the XML parsing should be in the `from_xml` method so that the constructor for the actual element takes its field types. So the parsing is in `from_xml` and then a normal existing constructor is used to instantiate the object.